frr/rootfs/entrypoint.sh

```
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+sed -i 's/^#?frr_global_options=.*/frr_global_options="-f /etc/frr/frr.${ROUTER}.conf
```

Shellcheck (0.10.0)

[error] Couldn't parse this single quoted string. Fix to allow more checks.

(SC1073)